pkg/yurthub/filter/interfaces.go

```diff
@@ -59,4 +62,35 @@ type ObjectFilter interface {
 	Filter(obj runtime.Object, stopCh <-chan struct{}) runtime.Object
 }
 
+type FilterManager interface {
```

Comment:
why define a new Interface for Filters?

Reply:
An interface is added to facilitate writing the UT of multiplexerProxy. When writing UT, a simple implementation can be made for Filtermanager.